exonum-light-client/src/main/java/com/exonum/client/ExplorerApiHelper.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2019 The Exonum Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.exonum.client;
+
+import static com.exonum.binding.common.serialization.json.JsonSerializer.json;
+
+import com.exonum.binding.common.hash.HashCode;
+import com.google.common.annotations.VisibleForTesting;
+import com.google.gson.annotations.SerializedName;
+import lombok.Value;
+
+/**
+ * Utility class for Exonum Explorer API.
+ */
+final class ExplorerApiHelper {
+
+  static String submitTxBody(String message) {
+    SubmitTxRequest request = new SubmitTxRequest(message);
+    return json().toJson(request);
+  }
+
+  static HashCode submitTxParser(String json) {
```

Comment:
`+parseSubmitTxResponse(String response)` (so that it is a verb describing the action, otherwise it sounds as it returns some kind of a parser)?